components/file_system/src/lib.rs

```diff
@@ -45,20 +49,20 @@ pub enum IOOp {
 #[repr(C)]
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, EnumCount)]
 pub enum IOType {
-    Other,
+    Other = 0,
     // Including coprocessor and storage read.
-    ForegroundRead,
+    ForegroundRead = 1,
     // Including scheduler worker, raftstore and apply. Scheduler worker only
     // does read related works, but it's on the path of foreground write, so
     // account it as foreground-write instead of foreground-read.
-    ForegroundWrite,
-    Flush,
-    Compaction,
-    Replication,
-    LoadBalance,
-    Gc,
-    Import,
-    Export,
+    ForegroundWrite = 2,
+    Flush = 3,
+    Compaction = 4,
+    Replication = 5,
+    LoadBalance = 6,
+    Gc = 7,
```

Comment:
I'm not familiar with that, but it seems that filter only makes decisions without issuing IOs? It would be unfair to tag the entire compaction as GC just for a few keys being GCed.

Reply:
The compaction filter only filter write CF, then use the info and send a GC task to another thread pool to GC the default CF. The GC task is done by writing rocksdb.

Reply:
So these GC tasks are scheduled in normal gc threads? Then they should be tagged by `GcRunner`.